podium-gateway/src/main/java/nl/thehyve/podium/enumeration/RequestFileType.java

```diff
@@ -0,0 +1,16 @@
+package nl.thehyve.podium.enumeration;
+
+
+/**
+ * The RequestStatus enumeration.
+ */
+public enum RequestFileType {
+    METC_LETTER,
+    ORG_CONDITIONS,
+    MTA,
+    DTA,
+    OTHER,
+    NOT_SET,
```

Comment:
Trailing comma. Convention for not set is `None`. Maybe document the options and try to prevent using TLA (Three letter acronyms).